lib/adhearsion/version.rb

```diff
@@ -1,5 +1,5 @@
 # encoding: utf-8
 
 module Adhearsion
-  VERSION = '2.6.2'
+  VERSION = '2.6.3'
```

Comment:
Please don't bump the version, just add the changelog entry to the `develop` header.